tests/suites/test_suite_psa_crypto.data

```diff
@@ -1326,19 +1382,19 @@ PSA symmetric encryption multipart: AES-CTR, 11+5 bytes [#2]
 depends_on:MBEDTLS_AES_C:MBEDTLS_CIPHER_MODE_CTR
 cipher_decrypt_multipart:PSA_ALG_CTR:PSA_KEY_TYPE_AES:"2b7e151628aed2a6abf7158809cf4f3c":"2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a":"6bc1bee22e409f96e93d7e117393172a":11:11:5:"8f9408fe80a81d3e813da3c7b0b2bd32"
 
-PSA symmetric encryption multipart: AES-CTR, 16+16 bytes [#2]
+PSA symmetric decryption multipart: AES-CTR, 16+16 bytes [#2]
```

Comment:
Thanks for noticing and fixing the copypasta. Now that it's fixed, the `[#2]` suffix, and `[#1]` on the actual encryption test cases, can be removed. The suffixes `[#NUMBER]` were added automatically to make test case descriptions unique for reporting purpose, and fixing the duplicates (sometimes due to copypasta as here, sometimes due to descriptions that aren't precise enough) is an [ongoing task](https://github.com/ARMmbed/mbed-crypto/issues/271).

Reply:
Sure.

Reply:
Done.